firmware/controllers/can/can_dash.h

```diff
@@ -7,10 +7,21 @@
 
 #pragma once
 
-void canDashboardBMW();
-void canDashboardFiat();
-void canDashboardVAG();
-void canMazdaRX8();
-void canDashboardW202();
-void canDashboardBMWE90();
-void canDashboardVagMqb();
+#define CAM_5ms         (1<<0)
```

Comment:
is `CAM` correct?

Reply:
CAM - is short for Cyclic Activity Manager, i can change it, it cam to me natural since using same abbreviation on other projects.

Reply:
> using same abbreviation on other projects

I would be very careful mixing professional life and hobby life, if I'm reading this correctly.